plugins/Bump/class.bump.plugin.php

```diff
@@ -13,7 +13,7 @@ public function base_discussionOptions_handler($sender, $args) {
         $discussion = $args['Discussion'];
         if (checkPermission('Garden.Moderation.Manage')) {
             $label = t('Bump');
-            $url = url("discussion/bump?discussionid={$discussion->DiscussionID}");
+            $url = "discussion/bump?discussionid={$discussion->DiscussionID}";
```

Comment:
By not wrapping this at all you may be causing a problem downstream, when a totally unqualified url gets set in the Args

https://github.com/vanilla/addons/blob/69468c31e252978ca16e58d8d32d463fcf1f3a7f/plugins/Bump/class.bump.plugin.php#L23

The better solution here is likely to still use the `url()` function, but to pass true as the second parameter so it becomes an absolute url. This will prevent subsequent calls from breaking the url.